subprojects/rzheap/rz_jemalloc/internal/bitmap.h

```diff
@@ -2,83 +2,82 @@
 #ifdef JEMALLOC_H_TYPES
 
 /* Maximum bitmap bit count is 2^LG_BITMAP_MAXBITS. */
-#define	LG_BITMAP_MAXBITS	LG_RUN_MAXREGS
-#define	BITMAP_MAXBITS		(ZU(1) << LG_BITMAP_MAXBITS)
+#define LG_BITMAP_MAXBITS LG_RUN_MAXREGS
```

Comment:
Also this change probably better as a separate PR

Reply:
This is actually not necessary... my editor did more than necessary here.